examples/seismic/model.py

```diff
@@ -461,3 +462,33 @@ def vp(self, vp):
             initialize_function(self.m, 1 / (self.vp * self.vp), self.nbpml)
         else:
             self.m.data = 1 / vp**2
+
+    def __getstate__(self):
+        state = dict(self.__dict__)
+        state.pop('grid')
+        state.pop('damp')
+        state.pop('m')
+        state.pop('_vp')
+        dim = len(self.shape)
+        if dim == 1:
```

Comment:
Now that I think of it, I don't think this one is supported anywhere so far. Good reminder.